portfolio/src/main/webapp/script.js

```diff
@@ -18,9 +18,14 @@
 function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
+  
+  const quotes = 
+    ['They may take our lives, but they\'ll never take our freedom', 
+     'Chewie, we\'re home', 'Just when I thought I was out, they pull me back in',
```

Comment:
One fairly common practice is to use double quotes if the string contains a single quote and vice versa (so use "Chewie, we're home" instead of 'Chewie we\'re home').

This works for languages that support both like Python & Javascript, but not Java or C++.